src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -0,0 +1,315 @@
+                   type="text"
+                   placeholder="Name..."
+                   id="name"
+                   name="name"
+                   formControlName="name"
+                   autofocus>
+            <span i18n
+                  class="help-block"
+                  *ngIf="(formDir.submitted || rbdForm.controls.name.dirty) && rbdForm.controls.name.hasError('required')">
+              This field is required.
+            </span>
+          </div>
+        </div>
+
+        <!-- Pool -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || rbdForm.controls.pool.dirty) && rbdForm.controls.pool.invalid}">
+          <label class="control-label col-sm-3"
+                 for="pool">
+            Pool
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <select id="pool"
+                    name="pool"
+                    class="form-control"
+                    formControlName="pool">
+              <option *ngIf="pools === null"
+                      [ngValue]="null">Loading...
+              </option>
+              <option *ngIf="pools !== null && pools.length === 0"
+                      [ngValue]="null">-- No pools available --
+              </option>
+              <option *ngIf="pools !== null && pools.length > 0"
+                      [ngValue]="null">-- Select a pool --
+              </option>
+              <option *ngFor="let pool of pools"
+                      [value]="pool.pool_name">{{ pool.pool_name }}</option>
+            </select>
+            <span i18n
+                  class="help-block"
+                  *ngIf="(formDir.submitted || rbdForm.controls.pool.dirty) && rbdForm.controls.pool.hasError('required')">
+              This field is required.
+            </span>
+          </div>
+        </div>
+
+        <!-- Use a dedicated pool -->
+        <div class="form-group">
+          <div class="col-sm-offset-3 col-sm-9">
+            <div class="checkbox checkbox-primary">
+              <input type="checkbox"
+                     id="useDataPool"
+                     name="useDataPool"
+                     formControlName="useDataPool">
+              <label i18n
+                     for="useDataPool">Use a dedicated data pool</label>
+            </div>
+          </div>
+        </div>
+
+        <!-- Data Pool -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || rbdForm.controls.dataPool.dirty) && rbdForm.controls.dataPool.invalid}"
+             *ngIf="rbdForm.value.useDataPool">
+          <label class="control-label col-sm-3"
+                 for="dataPool">
+            Data pool
+            <span class="required"></span>
+            <cd-helper html="Dedicated pool that stores the object-data of the RBD.">
+            </cd-helper>
+          </label>
+          <div class="col-sm-9">
+            <select id="dataPool"
```

Comment:
Nit: might be nice if the data pool drop-down automatically excludes the currently selected image pool.